project/Merging.scala

```diff
@@ -7,7 +7,7 @@ object Merging {
     case PathList("google", "protobuf", _ @ _*) => MergeStrategy.first
     case PathList("META-INF", "versions", "9", "module-info.class") => MergeStrategy.first
     case "module-info.class" =>
-      MergeStrategy.discard // JDK 8 does not use the file module-info.class so it is safe to discard the file.
+      MergeStrategy.discard
```

Comment:
is this safe for JDK 11 too?  should this note still stand?

Reply:
I believe this is safe. I was surprised that it was, but tests/compilation fail if this isn't here, and succeeds if it is, so I think the original comment may have been wrong? Or I am misunderstanding.